lib/ur/failurereporting.go

```diff
@@ -192,7 +194,7 @@ func sendFailureReports(ctx context.Context, reports []FailureReport, url string
 
 	reqCtx, reqCancel := context.WithTimeout(ctx, sendTimeout)
 	defer reqCancel()
-	req, err := http.NewRequestWithContext(reqCtx, http.MethodGet, url, &b)
+	req, err := http.NewRequestWithContext(reqCtx, http.MethodPut, url, &b)
```

Comment:
Technically I think this ought to be a POST ("send data/command to this resource") rather than PUT ("replace the content of this resource")?